app/services/bibliography_builder.rb

```diff
@@ -39,6 +39,7 @@ def traject_indexer
   private
 
   def convert_id(doc)
+    return nil unless doc['id']
```

Comment:
`nil` is redundant in this return I believe.

Reply:
Thanks for catching that. I've pushed another commit that updates it since #633 is relying on one of these commits now.